common/djangoapps/student/tests/tests.py

```diff
@@ -15,6 +15,7 @@
 from django.test.client import Client
 from mock import Mock, patch
 from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from urlparse import urljoin
```

Comment:
Minor: `urlparse` is part of the standard library; this belongs with the imports above. `pytz` is a third-party package and should be moved.